pwndbg/commands/heap.py

```diff
@@ -113,7 +113,7 @@ def heap(addr=None, verbose=False):
     # heap's mapping, otherwise, compensate for the presence of a heap_info
     # struct and possibly an arena.
     if addr:
-        cursor = int(addr)
+        cursor = int(addr) - 2 * ptr_size
```

Comment:
This line assumes a chunk's user data address was passed in the `address` argument, whereas the help field for the `address` argument indicates it should be the start of a `malloc_chunk` struct. The command still works when you pass the address of a `malloc_chunk` struct because of the i686 alignment heuristic shown below:
https://github.com/pwndbg/pwndbg/blob/5062e4afce533692eaea79488ff316b3a688b17f/pwndbg/commands/heap.py#L127-L130

Reply:
Good spot!
I was wondering if heap related commands should receive start of the struct or addr of `fd` field and this is missed prototyping ;) thx